tests/camel-itest-spring-boot/src/test/resources/components/bam-persistence.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2006 The Apache Software Foundation.
```

Comment:
Can you use the same license header as the other XML files in Apache Camel. They need to be similar as we have a RAT check that check the code for license alignments.